docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+When setting up an uptime monitor, you will use a provided Docker image to run the monitor and send data back to Elasticsearch. 
+There are two approaches to configuring the monitor and connecting it to Elasticsearch.
+
+After set up is complete, the data will be available in the {uptime-app} in {kib} to view, analyze, and alert on.
+
+[discrete]
+[[uptime-set-up-choose]]
+== Choose an approach
+
+There are two ways to configure uptime monitors and connect them to the {stack}:
+
+* Use a *{heartbeat}* configuration file
```

Comment:
Shouldn't we just say: "use heartbeat"? I mean that because you're mentioning a HB config file and Agent, not HB config file and Agent Config or anything of that kind (and HB is what actually runs the monitors, the config file just tells it what to run).